thunder/core/jit_ext.py

```diff
@@ -905,7 +905,17 @@ def general_jit_lookaside(fn, *args, **kwargs) -> None | Callable:
         def is_from_torch(fn):
             return hasattr(fn, "__module__") and fn.__module__ and fn.__module__.startswith("torch")
 
-        if is_opaque(fn) and is_from_torch(fn):
+        has_tensor_arg = False
+        for a in args:
+            if isinstance(a.value, TensorProxy):
+                has_tensor_arg = True
+                break
+            if isinstance(a.value, Sequence):
```

Comment:
This peeks only one layer. Then again, I don't know of any torch functions that take more than one layer of wrapped sequences.

Reply:
Last I looked at native functions, they did not, which is why I implemented it this way.